stmhal/usbdev/class/inc/usbd_cdc_msc_hid.h

```diff
@@ -20,6 +20,12 @@
 #define CDC_OUT_EP    (0x03)
 #define CDC_CMD_EP    (0x82)
 
+// Need to define here for usbd_hid_interface.c (it needs HID_OUT_EP_WITH_CDC)
+#define HID_IN_EP_WITH_CDC (0x81)
+#define HID_OUT_EP_WITH_CDC (0x01)
+#define HID_IN_EP_WITH_MSC (0x83)
+#define HID_OUT_EP_WITH_MSC (0x03)
```

Comment:
These shouldn't be exposed.  Instead one should use the `hid_out_ep` variable defined in usbdev/class/src/usbd_cdc_msc_hid.c.  Two options to make it work: 1) make a function in usbd_cdc_msc_hid.c to get the hid out EP (eg USBD_HID_GetOutEP); 2) add functions to usbd_cdc_msc_hid.c to set the SNAK/CNAK bits and call those functions from usbd_hid_interface.c.  Probably the latter is cleaner.